src/lib.rs

```diff
@@ -458,6 +462,43 @@ trait Signer {
     fn key(&self) -> &Key;
 }
 
+impl<'a> JoseNewAccount<'a> {
```

Comment:
Style: this should go underneath the defintion of `JoseNewAccount` (if we keep that).